packages/perseus/src/widgets/radio/radio-validator.ts

```diff
@@ -21,23 +23,33 @@ function radioValidator(
         };
     }
 
-    // TODO(LEMS-2541): should numCorrect actually be on the rubric
-    // instead of the userInput?
-    if (
-        userInput.numCorrect &&
-        userInput.numCorrect > 1 &&
-        numSelected !== userInput.numCorrect
-    ) {
+    const numCorrect: number = _.reduce(
+        rubric.choices,
+        function (memo, choice) {
+            return choice.correct ? memo + 1 : memo;
+        },
+        0,
+    );
+
+    if (numCorrect && numCorrect > 1 && numSelected !== numCorrect) {
```

Comment:
```suggestion
    if (numCorrect > 1 && numSelected !== numCorrect) {
```

I know you're just moving existing code, but `numCorrect` is a number and the only falsy number is 0 which is not greater than 1.

Reply:
Fixed!